src/components/timestampcontrol/TimestampControlDirective.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  goog.provide('ga_timestamp_control_directive');
+
+  var module = angular.module('ga_timestamp_control_directive', []);
+
+  module.directive('gaTimestampControl', function($rootScope) {
+    return {
+      restrict: 'A',
+      scope: {
+        map: '=gaTimestampControlMap'
+      },
+      templateUrl: 'components/timestampcontrol/partials/timestampcontrol.html',
+      link: function(scope, element) {
+        scope.layers = scope.map.getLayers().getArray();
+        scope.$watchCollection('layers', function(layers) {
```

Comment:
We could create an additional layer filter for geojson layers here: https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/map/MapService.js#L1174